js/flightlog_fields_presenter.js

```diff
@@ -431,12 +431,22 @@ function FlightLogFieldPresenter() {
 
         DEBUG_FRIENDLY_FIELD_NAMES = {...DEBUG_FRIENDLY_FIELD_NAMES_INITIAL};
 
+        if (firmwareType === FIRMWARE_TYPE_BETAFLIGHT && semver.gte(firmwareVersion, '4.2.0')) {
```

Comment:
We could just return on `if (firmwareType ...` once and only do the `semver` checks later on.

Reply:
Great, thanks.  Refactored.